src/main/java/com/conveyal/r5/transit/TransportNetwork.java

```diff
@@ -37,55 +38,60 @@
     public TransitLayer transitLayer;
 
     /**
-     * Cache a grid point set that covers the extents of this transport network. The PointSet itself caches linkages to
-     * street networks.
-     *
-     * Eventually we should save this point set and linkage but it's a little tricky since there's a linkage cache with
-     * references to streetlayers, a spatial index, etc. in there.
+     * A grid point set that covers the full extent of this transport network. The PointSet itself then caches linkages
+     * to street networks (the baseline street network, or ones with various scenarios applied). If they have been
+     * created, this point set and its linkage to the street network are serialized along with the network, which makes
+     * startup much faster. Note that there's a linkage cache with references to streetlayers in this GridPointSet,
```

Comment:
It doesn't have a reference to streetlayers, right? I mean, you still don't want to serialize it because it will contain too many linkages, but it won't pull in a bunch of streetlayers.